hardware/raspberrypi/power/README.md

```diff
@@ -2,7 +2,7 @@
 
 The Raspberry Pi 3 is powered by a +5.1V micro USB supply. Exactly how much current (mA) the Raspberry Pi requires is dependent on what you connect to it. We have found that purchasing a 2.5A power supply from a reputable retailer will provide you with ample power to run your Raspberry Pi. You can purchase the [official Raspberry Pi Power Supply](https://www.raspberrypi.org/products/raspberry-pi-universal-power-supply/) from our website, and you can learn more about the differing power requirements of the various models of the Raspberry Pi on our [FAQ page](https://www.raspberrypi.org/help/faqs/#power). 
 
-Typically, the model B uses between 700-1000mA depending on what peripherals are connected; the model A can use as little as 500mA with no peripherals attached. The maximum power the Raspberry Pi can use is 1 Amp. If you need to connect a USB device that will take the power requirements above 1 Amp, then you must connect it to an externally-powered USB hub.
+Typically, the model B uses between 700-1000mA depending on what peripherals are connected; the model A can use as little as 500mA with no peripherals attached. The maximum USB power the Raspberry Pi peripherals can use is 1 Amp. If you need to connect a USB device that will take the USB power requirements above 1 Amp, then you must connect it to an externally-powered USB hub.
```

Comment:
I think "The maximum USB power the Raspberry Pi peripherals can use is 1 Amp." would be clearer as "The maximum power the Raspberry Pi USB peripherals can use is 1 Amp."?